perfdash/parser.go

```diff
@@ -258,32 +258,34 @@ type schedulingMetrics struct {
 	FrameworkExtensionPointDuration map[string]latencyMetric `json:"frameworkExtensionPointDuration"`
 }
 
-func parseOperationLatency(latency latencyMetric, operationName string) perftype.DataItem {
-	perfData := perftype.DataItem{Unit: "ms", Labels: map[string]string{"Operation": operationName}, Data: make(map[string]float64)}
+func parseOperationLatency(latency latencyMetric, testName string, operationName string) perftype.DataItem {
+	perfData := perftype.DataItem{Unit: "ms", Labels: map[string]string{"TestName": testName, "Operation": operationName}, Data: make(map[string]float64)}
```

Comment:
Will adding the TestName label in future parses of "density" workloads affect the graphs? Or should I default to not adding the TestName label if `testName == "density"` to keep DataItems compatible with old data?

Reply:
Parsed data is not stored everywhere. If we deploy a new perf-dash version it will parse all runs and build the internal data from scratch. Thus, I think it should work, but it'd be good to double check by running the perf-dash locally for this change - https://github.com/kubernetes/perf-tests/tree/master/perfdash#local-development

Reply:
![Screenshot_2020-09-03 Screenshot](https://user-images.githubusercontent.com/7521600/92136785-72a08f80-edda-11ea-8bd3-49b874673dec.png)

Local instance runs fine, but since only the density test exists right now under the kubemark jobs, there isn't any data. Once https://github.com/kubernetes/test-infra/pull/19044 is merged and once a few executions of that job complete, I'll check again and report back.